testdata/configtest.json

```diff
@@ -1,5 +1,6 @@
 {
```

Comment:
Revert this please. In positive news, once you update branch to latest in master, the test that used to always force a save of this file no longer does this.

Reply:
+